schema/table.go

```diff
@@ -174,6 +174,10 @@ func (t *Table) processFields(typ reflect.Type, canAddr bool) {
 				}
 			}
 
+			if subtable.Unique != nil && t.Unique == nil {
+				t.Unique = subtable.Unique
+			}
+
```

Comment:
I think we should add the Unique constraint when adding the embeded field later Line 246